### PR TITLE
publish 2.16.0 m3 update site

### PR DESCRIPTION
**Pull request by cdietrich**
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: milestones/compositeArtifacts.xml [hunk 1]
```diff
@@ -2,9 +2,9 @@
 <?compositeArtifactRepository version='1.0.0'?>
 <repository name='TMF Xtext Update Site (Milestones)' type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' version='1.0.0'>
   <properties size='1'>
-    <property name='p2.timestamp' value='1539639946675'/>
+    <property name='p2.timestamp' value='1543222972288'/>
   </properties>
   <children size='1'>
-    <child location='head/S201810150206'/>
+    <child location='head/S201811260210'/>
```
Comment (kthoms) [inline milestones/compositeArtifacts.xml:9]: shall we drop M1?

Comment (cdietrich) [reply milestones/compositeArtifacts.xml:9]: i dont know how this was done in the past.?!? do you remember?

Comment (cdietrich) [reply milestones/compositeArtifacts.xml:9]: am not sure if we catch all errors if p2 can fetch stuff from old update sites

Comment (kthoms) [reply milestones/compositeArtifacts.xml:9]: Usually the milestones are kept during a release, sometimes also longer. That's why it is a composite and does not hold last MS only.
Effectively noone will use M1 anymore. I'm fine with both.

**Comment by cdietrich** [comment]
@kthoms updated the PR

**Review by szarnekow — APPROVED** [review]

**Pull request merged**